MissionFollowmeTarget followme_target
---
bool result
